config/src/config/storage_config.rs

```diff
@@ -112,7 +116,10 @@ impl Default for StorageConfig {
             storage_pruner_config: StoragePrunerConfig {
                 state_store_prune_window: Some(1_000_000),
                 ledger_prune_window: Some(10_000_000),
-                pruning_batch_size: 500,
+                ledger_pruning_batch_size: 500,
+                // A 10k transaction block (touching 60k state values, in the case of the account
+                // creation benchmark) on a 4B items DB (or 1.33B accounts) yields 300k JMT nodes
+                state_store_pruning_batch_size: 300_000,
```

Comment:
I feel this value (~150M in size) is too large. Should we further split it into multiple commits? @msmouse WDYT?

Reply:
Yeah the idea is to make the batches smaller

Reply:
The intention was to keep is similar to the status quo and after I do some experiment, I will change this to a smaller value.

Reply:
Make this smaller to 1k.